### DISCUSSION TITLE
"MIDI2LR Quit Unexpectedly" every time I quit LR Classic

### DISCUSSION THREAD

**Discussion by J-Elledge** [rsjaffe/MIDI2LR#654] [Problems]
This isn't a huge deal; more of an annoyance. But every time I quit LR Classic, I get an error popup from MIDI2LR telling me that it quit unexpectedly. Am I doing something wrong? Is there a way around this?

**Comment by rsjaffe** [comment]
Could you post MIDI2LR.log?

**Reply by J-Elledge** [reply]
Sure. I apologize, but I cannot figure out where to locate the log.

**Reply by rsjaffe** [reply]
See https://github.com/rsjaffe/MIDI2LR/wiki#support

**Reply by J-Elledge** [reply]
Thank you for the reply and the help. For whatever reason, I don't have an option to generate diagnostic report. That menu (Help->Plug-in Extras->MIDI2LR shows Online help, Support, View latest release, Donate, and About. Apologies if I'm missing something very obvious.

**Reply by rsjaffe** [reply]
What version Lightroom are you using. What version MIDI2LR. What operating system version.

**Reply by J-Elledge** [reply]
LR Classic 11.3.1/ MIDI2LR 5.1.0.0/ OSX 12.3.1

**Reply by rsjaffe** [reply]
OK. That problem was fixed with the very next release, and there've been several releases since then. I'd update to the latest: https://github.com/rsjaffe/MIDI2LR/releases/latest

**Reply by J-Elledge** [reply]
Duh, don't know how I missed that. Thanks!
